nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/statuses/StatusServlet.java

```diff
@@ -86,12 +105,20 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
             } else {
                 resp.setStatus(HttpServletResponse.SC_SERVICE_UNAVAILABLE);
             }
+        } else if (requestedInfo.equals(PROBE_PARAM)) {
+            String probetoEval = req.getParameter(PARAM_SUMMARY_KEY);
+            HealthCheckResult result = getOrRunHealthCheck(probetoEval);
+            if (result.toJson().isEmpty()) {
```

Comment:
We can't rely on the fact that the string is empty to guess that the probe has not been found. The `getOrRunHealthCheck` has to send an exception that can be cought here.